memory/dma_memtest.py

```diff
@@ -176,8 +180,9 @@ def test(self):
             self.log.info('DMA memory test passed.')
 
     def tearDown(self):
-        self.log.info('Cleaning up')
-        for j in range(self.sim_cps):
-            tmp_dir = 'linux.%s' % j
-            shutil.rmtree(tmp_dir)
-        shutil.rmtree(self.base_dir)
+        if cancel_flag == 1:
```

Comment:
should it  not be ?
```
if cancel_flag == 0:

```

Reply:
tmp_dir = 'linux.%s' % j
if os.path.exists(tmp_dir):
             shutil.rmtree(tmp_dir)

Reply:
Done